tests/appsec/test_user_blocking_full_denylist.py

```diff
@@ -33,7 +33,6 @@ def setup_blocking_test(self):
 
     @bug(context.library < "ruby@1.12.1", reason="APMRP-360")
     @bug(context.library >= "java@1.22.0" and context.library < "java@1.35.0", reason="APMRP-360")
-    @bug(library="java", reason="APPSEC-56006")
```

Comment:
Could you set the version where it has been fixed ? If the fix is ready for version `1.2`, then it`s : 

```python
@bug(context.library < "java@1.2", reason="APPSEC-56006")
```

Reply:
Restored tag.

Reply:
Fix is starting for version `1.36.0` which is included in `@bug` annotation above